### DISCUSSION TITLE
What is the difference between Theme Make Current and Enable

### DISCUSSION THREAD

**Discussion by smemis** [OrchardCMS/OrchardCore#13500] [General]
<img width="543" alt="image" src="https://user-images.githubusercontent.com/4637285/229182830-8dfdf15d-fe45-4a2c-9c89-38d897b3b091.png">

and what does "Enable" do?

**Comment by ns8482e** [comment]
Just click on make current to activate the theme on front end

**Reply by smemis** [reply]
Always accessible all themes files on frontend without enable or disable.

when 
Enable button active I can access  /Mestech.Theme/assets/img/logo.png file
or 
Disable button active I can access /Mestech.Theme/assets/img/logo.png file.

I don't understand what does "Enable" do?

Make Current button activate a theme and execute recipe file if the "issetuprecipe": false 
but "issetuprecipe": true recipe file does not execute.

How can i use "Make Current" and "Enable/Disable" buttons?
What do these buttons do?

**Reply by ns8482e** [reply]
I believe only enabled themes are displayed in the theme's list. Themes are like modules that you can enable or disable.

However there is no significant benefit of enabled themes unless that becomes current. As non-current themes are ignored and not applied during theme selection on fronted.

**Reply by myetginbal** [reply]
> However there is no significant benefit of enabled themes unless that becomes current. As non-current themes are ignored and not applied during theme selection on fronted.

In this case, one may make use that status info to present alternatives in a theme selector or something like that in the front end for end user personalization. By doing so, admin will have the right the choose the ones that will be listed in the theme selector.

**Reply by ns8482e** [reply]
Also when You programmatically select the theme by overriding current theme - theme needs to be enabled
